### ISSUE TITLE
results: Show accessibility and places in a 15-minute walking period

### ISSUE THREAD

**Issue by tahini** [chairemobilite/localisation#46]
While the accessibility in transit is interesting for general housing choice, the 15-minutes city is a quite well-known concept and an important factor in the transportation behavior. It should be possible to see this accessibility, with detailed data as well. And also show the actual places (service type, exact location, name) on the map when zooming in.

The data does not necessarily have to be in the person's interview `response` field. This can come from a service from the backend.

**Comment by samuel-duhaime** [comment]
It's done, but it's quite simple for now. We show a circle. Should we close?

**Comment by greenscientist** [comment]
@samuel-duhaime We should the circle, but we don't show the places statistics I think? We should probably keep open
